web/service/src/main/java/org/mobilitydata/gtfsvalidator/web/service/controller/ValidationController.java

```diff
@@ -121,6 +127,22 @@ public ResponseEntity runValidator(
     }
   }
 
+  @GetMapping("/version")
+  public ResponseEntity currentVersion() {
```

Comment:
Let's annotate the ResponseEntity with the right response class. This change will propagate some changes in the return statement. 
```suggestion
  public ResponseEntity<String> currentVersion() {
```